frontend/public/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "React App",
```

Comment:
we should use project data instead of example placeholders :wink:

Reply:
thanks! fixed!